docs/README_RECON.md

```diff
@@ -172,12 +295,31 @@ class Thresholds:
 
 ### filters
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
 @dataclass
 class Filters:
     source: str | None = None
     target: str | None = None
-```
+</pre>
+</td>
+<td>
+<pre lang="json">
+"filters":{
+  "source": "p_price > 1000",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements.

Reply:
Fixed